routers/favorite_teams/favorite_teams-model.js

```diff
@@ -37,6 +37,7 @@ function insertFavoriteTeamByUser(team) {
 
 function removeFavoriteTeamByUser(id) {
   return db("favorite_teams")
-    .where("favorite_id", id)
+    .where("team_id", id)
```

Comment:
Im not sure this is going to work. Wont it delete all the teams that use that team_id ?